templates/client-config-secret.conf.j2

```diff
@@ -57,3 +57,5 @@ enable_v1 = true
 
 [platform:aarch64]
 architecture = arm64
+
+auto_build_node_selector = {{ osbs_auto_build_node_selector }}
```

Comment:
This should be conditional, only set it if value is truthy.
More importantly, this is being defined under the wrong config section.

We need to move this over so it's under the `[{{ cluster.name }}]` section(s).
Thinking about it, this should actually be defined per cluster (`cluster.auto_build_node_selector`).

Reply:
ok made it defined per cluster